<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceMeasureCopyPropertyDialog</class>
 <widget class="QDialog" name="DistanceMeasureCopyPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance Measure Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="startPointLabel">
       <property name="text">
        <string>Start Point:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="RealNumberEditWidget" name="startPointXEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="startPointCommaLabel">
       <property name="text">
        <string>, </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="RealNumberEditWidget" name="startPointYEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="endPointLabel">
       <property name="text">
        <string>End Point:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="RealNumberEditWidget" name="endPointXEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="endPointCommaLabel">
       <property name="text">
        <string>, </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="RealNumberEditWidget" name="endPointYEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="labelGroupBox">
     <property name="title">
      <string>Label</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showLabelCheckBox">
        <property name="text">
         <string>Show Label</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Content</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
         <item row="0" column="0">
          <widget class="QRadioButton" name="labelAutoRadioButton">
           <property name="text">
            <string>Auto</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="autoLabelLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="labelCustomRadioButton">
           <property name="text">
            <string>Custom</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="customLabelLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Position</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QRadioButton" name="lpCenterTop">
             <property name="text">
              <string>Center-Top</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="lpMiddleLeft">
             <property name="text">
              <string>Middle-Left</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QRadioButton" name="lpMiddleRight">
             <property name="text">
              <string>Middle-Right</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="lpCenterBottom">
             <property name="text">
              <string>Center-Bottom</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="fontSizeLabel">
          <property name="text">
           <string>Font Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fontSizeSpinBox">
          <property name="minimum">
           <number>6</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showMarkersCheckBox">
     <property name="text">
      <string>Show End Markers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="markerSizeLabel">
       <property name="text">
        <string>Marker Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="markerSizeSpinBox">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorEditWidget" name="colorWidget">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/realnumbereditwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistanceMeasureCopyPropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DistanceMeasureCopyPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showLabelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showLabelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showMarkersCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>markerSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
